File: vim/templates/perl/kfhead.pl
################################################################################

=head2 FUNC( )

DESCRIPTION

B<In:>

	ARG_IN

B<Out:>

	ARG_OUT

B<Return:>

	M_OK
    M_ERR
    M_ERR_ARG

=cut
################################################################################
sub FUNC {
	my ( $ARG )		= @_;

	my ( $ret, $msg )	= ( M_OK, "" );
	
	_DBG($dbg, "{FUNC( )");
	
	unless ( $ARG )		_DBGERR( $dbg, M_ERR_ARG );
	

	$ret = M_OK;
	
DONE:
	_DBGRETURN( $dbg, $ret );	
}

File: vim/templates/c/kfhead.c
/*****************************************************************************
 * FNAME()
 *
 * AUTOR:
 * REVISION:
 *
 * BESCHREIBUNG
 *
 * (IN)
 * 		ARGNAME		DESCR
 * (OUT)
 * 		ARGNAME		DESCR
 * (RETURN VALUE)
 * 		VALUE		DESCR
 */
int FNAME( ARGS )
{
	int					ret				= CP_OK;

	_DBG(( dbg, "<FNAME( )" ));

	if ( !ARG )							_DBGERR( dbg, CP_ERR_ARG );

	ret = CP_OK;
DONE:
	_DBGDRETURN( dbg, ret );
}
